common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -991,76 +991,35 @@ def test_save_problem_submitted_no_randomize(self, rerandomize):
 
     def test_check_button_name(self):
 
-        # If last attempt, button name changes to "Final Check"
         # Just in case, we also check what happens if we have
         # more attempts than allowed.
         attempts = random.randint(1, 10)
         module = CapaFactory.create(attempts=attempts - 1, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         module = CapaFactory.create(attempts=attempts, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
```

Comment:
As mentioned above, I'd recommend simplifying this test point considerably since there is no more "Final Check", and the text on the button is always the same.